libs/features/creature/src/waypoint-data/waypoint-data.component.html

```diff
@@ -0,0 +1,98 @@
+<keira-top-bar [selected]="handlerService.selected" [selectedName]="handlerService.selectedName" [isNew]="handlerService.isNew" />
+
+<div class="container-fluid">
+  @if (editorService.loading) {
+    <span [translate]="'LOADING'"></span>
+  }
+
+  @if (editorService.form && !!editorService.loadedEntityId && !editorService.loading) {
+    <div>
+      <div class="content-block">
+        <keira-query-output [docUrl]="docUrl" [editorService]="editorService" (executeQuery)="save($event)" />
+      </div>
+      <div class="content-block">
+        <form [formGroup]="editorService.form" class="form-group edit-form">
+          <div class="row">
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+              <label class="control-label" for="point">Point</label>
```

Comment:
not sure if we want to put here some LABEL for translations (like `LOADING` above)

Reply:
Yes, no problem. The important thing is that everything else goes along and that they are the right files. I will see how to review the translations.

Reply:
I have no problem adding spans for translations, but I wouldn't actually translate the attributes. What I would translate, in any case, is some kind of helper, such as a placeholder, to tell the person what that field indicates. But for now, you can leave it there, just like the rest of the tables.